CHANGELOG.md

```diff
@@ -105,6 +84,32 @@ it will be removed; but as it won't be user-visible this isn't considered a brea
 
 ### Emissary-ingress and Ambassador Edge Stack
 
+- Change: Emissary-ingress can no longer be made to configure Envoy using the v2 xDS configuration
+  API; it now always uses the v3 xDS API to configure Envoy.  This change should be mostly invisible
+  to users, with two notable exceptions: It removes support for `protocol_version: v2`, and
+  `regex_type: unsafe`.
+  **No `protocol_version: v2`**: Support for specifying `protocol_version: v2`
+  in `AuthService`, `RateLimitService`, and `LogService` resources is removed.  These resources each
+  have a `protocol_version` field that controls whether Envoy speaks the `v2` transport API or the
+  `v3` transport API when speaking to that service.  Due to Envoy's removal of all v2 Envoy APIs,
+  the `v2`` value will no longer be supported.  Note that `protocol_version: v2` is was default in
+  all Emissary-ingress 2.x releases.  It is now necessary to explicitly say `protocol_version:
+  v3`.
+  Users who use these resource types but don't explicitly say `protocol_version: v3` will need
+  to adjust their service implementations to understand the v3 protocols, and then update
+  Emissary-ingress resources to say `protocol_version` before upgrading to Emissary-ingress
+  3.0.0.
+  **No `regex_type: unsafe`**: The `regex_type` field will be removed from the `ambassador`
+  `Module`, meaning that it will not be possible to instruct Envoy to use the <a
+  href="https://en.cppreference.com/w/cpp/regex/ecmascript">ECMAScript Regex</a> engine rather than
+  the default <a href="https://github.com/google/re2">RE2</a> engine.
+  Users who rely on the specific
+  ECMAScript Regex syntax will need to rewrite their regular expressions with RE2 syntax before
+  upgrading to Emissary-ingress 3.0.0.
+  Note that the `AMBASSADOR_ENVOY_API_VERSION` environment
+  variable is now a misnomer, as it no longer configures which xDS API version is used, but it still
+  affects what the default protocol used for a `TracingService` that points at Zipkin.
```

Comment:
Why is this still true? Isn't this completely removed in V3 as well and no longer is necessary?

Reply:
I don't know, we'll see.